packages/login/src/vaadin-login-form-mixin.js

```diff
@@ -96,4 +140,11 @@ export const LoginFormMixin = (superClass) =>
     _onForgotPasswordClick() {
       this.dispatchEvent(new CustomEvent('forgot-password'));
     }
+
+    /** @private */
+    __setCustomFields() {
+      this._customFields = [...this.$.vaadinLoginFormWrapper.children].filter((node) => {
```

Comment:
The way this property is used in the code gives the impression that it could be a simple getter, which would also eliminate the need for Mutation Observer.

Reply:
Good point, thanks. Updated.